tsconfig.json

```diff
@@ -19,6 +19,7 @@
     "skipLibCheck": true,
     "rootDir": ".",
     "baseUrl": ".",
+    "types": ["node"],
```

Comment:
Needed to address warnings associated with the use of Buffer which we experienced when attempting to avoid using the TCF core library from IAB TL. Other uses of Buffer generate a warning in our environments without the line being present. There may be better ways to address this however.

(!) Plugin typescript: @rollup/plugin-typescript TS2580: Cannot find name 'Buffer'. Do you need to install type definitions for node? Try `npm i --save-dev @types/node`.       
..\paf-mvp-core-js\src\query-string.ts: (6:10)

6   return Buffer.from(data).toString('base64');
           ~~~~~~

..\paf-mvp-core-js\src\query-string.ts: (9:10)

9   return Buffer.from(data, 'base64').toString('ascii');